internal/server/v1/optimus/routes.go

```diff
@@ -11,5 +11,6 @@ func Routes(optimusClient optimusv1beta1grpc.JobSpecificationServiceClient) func
 
 	return func(r chi.Router) {
 		r.Get("/projects/{project_name}/jobs/{job_name}", handler.findJob)
+		r.Get("/", handler.list)
```

Comment:
let's follow the convention here

```
single job - "/projects/{project_name}/jobs/{job_name}"
job list - "/projects/{project_name}/jobs"
```

project name is required so there is no point making it as querystring, let's make it as a route params instead.

@sudheerpal @ayushi0014